### DISCUSSION TITLE
VoLTE and VoWiFi carrier Digi Romania

### DISCUSSION THREAD

**Discussion by mosualin** [kyujin-cho/pixel-volte-patch#71] [Carrier-specific Experiences]
hello,
This patch works in the digi romanian network ( VoLTE and VoWiFi) MCC 226 MNC 05. I would like to ask you if you can please add in the table in the combatability matrix. "Compatibility matrix for known carriers" from here. 
https://github.com/kyujin-cho/pixel-volte-patch/blob/main/docs/compatibility-chart.en.md
and Digi Romania.

thx,

**Comment by ov1d1u** [comment]
I can confirm that it works for Digi.Mobil Romania.

**Comment by kyujin-cho** [comment]
Thanks for the report! I've updated the compat matrix.
